demos/datepicker/datepicker-fr.js

```diff
@@ -6,7 +6,7 @@
 	if ( typeof define === "function" && define.amd ) {
 
 		// AMD. Register as an anonymous module.
-		define([ "../jquery.ui.datepicker" ], factory );
+		define([ "datepicker" ], factory );
```

Comment:
The datepicker localization demo, is the only demo in the whole repo, that has JavaScript files that are not within the `ui/` so i think it makes sense to use this as the `baseUrl`.